test/controllers/tickets_controller_test.rb

```diff
@@ -70,6 +71,18 @@ class TicketsControllerTest < ActionController::TestCase
 
   end
 
+  test 'should accept tickets in Base64 encoding' do
+    I18n.locale = :nl
+
+    assert_difference 'ActionMailer::Base.deliveries.size', User.agents.count do
+      assert_difference 'Ticket.count' do
+        post :create, message: @simple_base64_email, format: :json
```

Comment:
@frenkel - the check in the controller looks to see if the base64 param is true, but also attempts to match it against a regex that should detect if it is base64 encoded already. In this case, that particular piece of text matches the regex, hence why the base64 param isn't required.